std/getopt.d

```diff
@@ -700,7 +713,16 @@ private void getoptImpl(T...)(ref string[] args, ref configuration cfg,
         else
         {
             // it's an option string
-            auto option = to!string(opts[0]);
+            static if (is(typeof(opts[0]) == string))
+            {
+                // allocate a new string so as to remove the
+                // scope restriction
```

Comment:
That really sucks. Safety shouldn't result in a performance overhead :/
Especially because in this case
- `getoptImpl` doesn't store anything, but modifies `configuration` and `rslt`
- it's a GC allocated string